tests/components/motion_blinds/test_config_flow.py

```diff
@@ -362,3 +367,52 @@ async def test_options_flow(hass):
     assert config_entry.options == {
         const.CONF_WAIT_FOR_PUSH: False,
     }
+
+
+async def test_change_connection_settings(hass):
+    """Test changing connection settings by issuing a second user config flow."""
+    config_entry = MockConfigEntry(
+        domain=const.DOMAIN,
+        unique_id=TEST_MAC,
+        data={
+            CONF_HOST: TEST_HOST,
+            CONF_API_KEY: TEST_API_KEY,
+            const.CONF_INTERFACE: TEST_HOST_HA,
+        },
+        title=DEFAULT_GATEWAY_NAME,
+    )
+    config_entry.add_to_hass(hass)
+
+    assert await hass.config_entries.async_setup(config_entry.entry_id)
+    await hass.async_block_till_done()
+
+    assert config_entry.data[CONF_HOST] == TEST_HOST
+    assert config_entry.data[CONF_API_KEY] == TEST_API_KEY
+    assert config_entry.data[const.CONF_INTERFACE] == TEST_HOST_HA
+
```

Comment:
```suggestion
```

We don't need to setup here.

The assert are testing the mock and not the integration so we should drop them.